Tests/SwiftStdlibTests/ArrayExtensionsTests.swift

```diff
@@ -36,6 +36,21 @@ final class ArrayExtensionsTests: XCTestCase {
         XCTAssertEqual(swappedEmptyArray, emptyArray)
     }
 
+    func testSortedLike() {
+        let expected1 = [1, 2, 3, 4, 5]
```

Comment:
Expected seems not the correct name for this, in tests it is normally the name used in the variable used in the assertion expressing the expected result, and since here it is the `like` parameter ... the expected name might be confusing.

Reply:
Try `order` instead